mne/preprocessing/ica.py

```diff
@@ -2168,15 +2200,17 @@ def run_ica(raw, n_components, max_pca_components=100,
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
+    method : {'fastica', 'picard'}
+        The ICA method to use. Defaults to 'fastica'.
```

Comment:
If we really believe your results, we should deprecate the default to `picard` (with a warning if it's not available that one should install it, and that we will fall back to `fastica`). We already do a similar warn-fallback with the mixed-norm solvers.

Reply:
Let's not get ahead of ourselves. FastICA has been around for a long time, and has been validated lots of times. Picard is very new, and I'd suggest adding it as an additional option is fine for now.

Furthermore, what happened to `'infomax'` and `'extended-infomax'` - shouldn't they be in this list?

Reply:
So the belief is not there :)

Reply:
Not yet 😄 - but we shouldn't rely on belief anyway, we're not talking about religion here 😉.